protocol/wlr-screencopy-unstable-v1.xml

```diff
@@ -102,6 +102,12 @@
 
         The client should then create a buffer with the provided attributes, and
         send a "copy" request.
+
+        When the buffer is a dmabuf, the stride parameter may be ignored.
+
+        The format parameter shall be interpreted according to
```

Comment:
Hmm, this isn't great. I wonder if we could do something better, like separate wl_shm and dmabuf hints…

Reply:
Yeah, this is a bit lame but the alternative is to have the user implement an almost identical callback with slightly different parameters. In the end, I believe this to be the more pragmatic approach.